tmva/tmva/test/DNN/GRU/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+############################################################################
+# CMakeLists.txt file for building TMVA/DNN/GRU tests.
+# @author Surya S Dwivedi
+############################################################################
+
+project(tmva-tests)
```

Comment:
@lmoneta, I wanted to ask why each directory in  tmva/tmva/test/DNN/[GRU, LSTM and others] is setting the same project "tmva-tests"? (it looks like copy-paste and is not needed at all). The same question is about next lines: 

`find_package(ROOT REQUIRED)
set(Libraries Core MathCore Matrix TMVA)
include_directories(${ROOT_INCLUDE_DIRS})` 
Are they really needed?